pkg/kubelet/eviction/helpers.go

```diff
@@ -93,8 +95,17 @@ func validSignal(signal evictionapi.Signal) bool {
 }
 
 // ParseThresholdConfig parses the flags for thresholds.
-func ParseThresholdConfig(evictionHard, evictionSoft, evictionSoftGracePeriod, evictionMinimumReclaim string) ([]evictionapi.Threshold, error) {
+func ParseThresholdConfig(includeAllocatableThreshold bool, evictionHard, evictionSoft, evictionSoftGracePeriod, evictionMinimumReclaim string) ([]evictionapi.Threshold, error) {
 	results := []evictionapi.Threshold{}
+	if includeAllocatableThreshold {
+		results = append(results, evictionapi.Threshold{
```

Comment:
Don't turn this on by default. Instead tie allocatable limits to `memory.available` signal.